README.md

```diff
@@ -1 +1,98 @@
-# personalPortfolio
\ No newline at end of file
+# personalPortfolio
+
+> This is a personal portfolio website that will be used to display my projects.
+
+
+
+## Built With
+
+- HTML5
+- CSS
+- Technologies used
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Good knowledge of Figma
+Create Github repository
+Have VSCode or choice text editor installed
+Have Git installed
+Good Knowledge of Linters
+
+### Setup
+Git clone - https://github.com/esterstarr/My-Portfolio
+HTML, CSS Linter configuration - https://github.com/microverseinc/linters-config/tree/master/html-css
+
+### Deployment
+Github
+
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/estherstarr)
+- Twitter: [@twitterhandle](https://twitter.com/anibeEsther)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/onwuanibeonome)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+
+
+
+
+
+
+
+
+# Personal Portfolio
+
+>  This is a personal portfolio website that will be used to display my projects.
+
+
+## Built With
+
+- HTML
+- CSS
+
+
+👤 **Author1***
+
+- GitHub: [@githubhandle](https://github.com/Estherstarr/)
+- Twitter: [@twitterhandle](https://twitter.com/AnibeEsther)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/onwuanibeonome)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
```

Comment:
- [ ] [OPTIONAL] As it was not marked by the previous review, this is marked as an optional suggestion: 
- Please, could you remove the doubling of information in README file 👍